extensions/google-chrome/src/components/ChromeListItems.tsx

```diff
@@ -9,10 +9,18 @@ export class ChromeListItems {
   public static TabHistory = HistoryItem;
 }
 
-function HistoryItem({ profile, entry: { url, title, id } }: { entry: HistoryEntry; profile: string }): ReactElement {
+function HistoryItem({
+  profile,
+  entry: { url, title, id },
+  type,
+}: {
+  entry: HistoryEntry;
+  profile: string;
+  type: "History" | "Bookmark";
```

Comment:
I'm unfamiliar with the implementation difference between bookmarks and history. It seems strange that bookmarks would use a history entry here. Are they so similar that bookmarks and history will always use the same logic?